docs/man/rpm-config.5.scd

```diff
@@ -0,0 +1,217 @@
+read sequentially by *rpm* for macro definitions. *%{\_target}* is
+expanded to the detected <arch>-<os> platform. Tildes are expanded
+to the value of the environment variable *HOME*.
+
+If a macro is defined multiple times, the last entry wins. The default
+macro path uses this to achieve the following hierarchy of settings:
+. Generic *rpm* factory defaults
+. Platform-specific *rpm* factory defaults
+. Vendor (distribution) specific settings
+. Host specific settings
+. User specific settings
+. Command-line override settings
+
+The default macro path can be inspected with *rpm --showrc|grep ^Macro*.
+
+In older versions of rpm, the path of per-user macros was _~/.rpmmacros_.
+This is still processed if it exists and the new configuration directory
+does not exist.
+
+# CONFIGURATION
+The following configurables are supported for the *rpm* runtime (as
+opposed opposed to just package building) parts:
+
+*%\_color_output* _MODE_
+	Output coloring mode. Valid values are *never* and *auto*.
+
+*%\_db_backend* _BACKEND_
+	The database backend to use. Possible values for _BACKEND_ are:
+	- *dummy* Dummy backend (no actual functionality)
+	- *bdb_ro* Berkeley DB (read-only)
+	- *ndb* Native database (no external dependencies)
+	- *sqlite* Sqlite database
+
+*%\_dbpath* _DIRECTORY_
+	The location of the rpm database file(s).
+
+*%\_excludedocs* _VALUE_
+	Boolean (i.e. 1 == "yes", 0 == "no") that controls whether files
+	marked as %doc should be installed.
+
+*%\_flush_io* _VALUE_
+	Flush file IO during transactions (at a severe cost in performance
+	for rotational disks). Possible values are 1 to enable, 0 to disable.
+
+*%\_group_path* _PATH_
+	Location of group(5) files as : separated list
+
+*%\_httpport* _PORT_
+	The port of HTTP proxy (used for FTP/HTTP).
+
+*%\_httpproxy* _HOSTNAME_
+	The hostname of HTTP proxy (used for FTP/HTTP).
+
+*%\_install_langs* _LOCALES_
+	A colon separated list of desired locales to be installed;
+	*all* means install all locale specific files.
+
+*%\_install_script_path* _PATH_
+	The PATH put into the environment before running %pre/%post et al.
+
+*%\_keyring* _BACKEND_
+	The keyring type to use. Possible values for _BACKEND_ are:
+	- *fs* Plain ASCII files in a directory
+	- *openpgp* Shared OpenPGP certificate directory
+	- *rpmdb* Pseudo-packages in the rpmdb
+
+*%\_keyringpath* _DIRECTORY_
+	The location of the keyring path for non-rpmdb variants.
+
+*%\_minimize_writes* _VALUE_
+	Minimize writes during transactions (at the cost of more reads) to
+	conserve eg SSD disks (EXPERIMENTAL). Possible values are:
+	- *0* disable
+	- *1* enable
+	- *-1* (or undefined) autodetect on platforms where supported,
+	  otherwise default to disabled
+
+*%\_netsharedpath* _PATH_
+	A colon separated list of paths where files should *not* be installed.
+	Usually, these are network file system mount points.
+
+*%\_passwd_path* _PATH_
+	Location of passwd(5) files as : separated list
+
+*%\_pkgverify_flags* _VSFLAGS_
```

Comment:
I'd add something like "See Verification flags for details" here... but then, we'd have to add the same to each of the `%_vsflags_*` macros below, at which point it becomes quite hysterical...

Reply:
Yep, that's a tricky thing. One possibility would be covering all these in the "Verification flags" subsection, but then it seems somehow inconsistent too.

Reply:
Yeah, I'd say just leave it as-is for now. The format is documented, that's important.